schema-engine/connectors/mongodb-schema-connector/src/client_wrapper.rs

```diff
@@ -9,11 +9,10 @@ use schema_connector::{ConnectorError, ConnectorResult};
 pub struct Client {
     inner: mongodb::Client,
     db_name: String,
-    preview_features: BitFlags<PreviewFeature>,
 }
 
 impl Client {
-    pub async fn connect(connection_str: &str, preview_features: BitFlags<PreviewFeature>) -> ConnectorResult<Client> {
+    pub async fn connect(connection_str: &str, _preview_features: BitFlags<PreviewFeature>) -> ConnectorResult<Client> {
```

Comment:
Should we remove the parameter as well?

Reply:
I would recommend to leave it here. Both the `mongodb-schema-connector` and the `sql-schema-connector` rely on preview features, and we'll likely need this again in the future. Removing this `preview_features` parameter (and its propagated calls) to add it again in the future just adds noise, imho.